lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -780,11 +785,26 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   end
 
   defp handle_media_event(%{type: :sdp_offer, data: data}, ctx, state) do
-    state = Map.put(state, :track_id_to_metadata, data.track_id_to_track_metadata)
+    state =
+      state
+      |> Map.put(:track_id_to_metadata, data.track_id_to_track_metadata)
+      |> Map.put(:track_id_to_bitrates, data.track_id_to_track_bitrates)
```

Comment:
I don't like the way RIDs are handled. I don't think there is necessarily any error here, but it's a little inconsistent across modules if the RID is a string or an atom. Can we convert the key to atoms either here or maybe even in `media_event.ex` and be done with it?